Cargo.toml

```diff
@@ -11,10 +11,12 @@ kelvin = "0.18"
 nstack = "0.4"
 lazy_static = "1.3.0"
 hades252 = { git = "https://github.com/dusk-network/hades252", version = "0.6.0" }
-dusk-plonk = {version = "0.2.0", features = ["trace-print"]}
+# Depends on unpublished changes - set version after jubjub bump
+dusk-jubjub = { git = "https://github.com/dusk-network/jubjub", branch = "vlopes11/10-dhke" }
```

Comment:
It's okay for development, but in order to merge we need to use only the `jubjub` version included in `dusk-plonk`. So if some feature are not published yet we need to publish before we can merge this PR.